internal/nomad/nomad.go

```diff
@@ -256,8 +265,8 @@ func handleEvaluationEvent(evaluations map[string]chan error, event *nomadApi.Ev
 // If a new allocation is received, onNewAllocation is called. If an allocation is deleted, onDeletedAllocation
 // is called. The pendingAllocations map is used to store allocations that are pending but not started yet. Using the
```

Comment:
This comment seems to be outdated, as there is no `pendingAllocations` map any longer.